src/server/apis/dash-data.ts

```diff
@@ -87,10 +88,10 @@ export class DashData {
             reviewersCount === 0) {
           outgoingPr.status = {type: 'NoReviewers'};
         } else if (outgoingPr.status.type === 'WaitingReview') {
-          outgoingPr.status.reviewers = [
+          outgoingPr.status.reviewers = unique([
```

Comment:
Could just do:

```ts
outgoingPr.status.reviewers = Array.from(new Set([
  ...outgoingPr.reviewRequests,
  ...outgoingPr.reviews.map((review) => review.author),
]))
```

Reply:
This also works:

```ts
outgoingPr.status.reviewers = [...new Set([
  ...outgoingPr.reviewRequests,
  ...outgoingPr.reviews.map((review) => review.author),
])]
```

Reply:
Very cool. I went with the frist since I think its little bit clearer.